src/main/java/com/deloitte/elrr/util/Mapper.java

```diff
@@ -0,0 +1,32 @@
+package com.deloitte.elrr.util;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+
+/**
+ * A singleton wrapper for jackson.databind.ObjectMapper configured
+ * for the serialization and deserialization of objects.
+ */
+public final class Mapper {
```

Comment:
i cant't find in the PR where a mapper is even used.

Reply:
Oh yeah, this can be removed, you added it for JSON stuff in the example but it is no longer used

Reply:
https://github.com/US-ELRR/elrrservices/pull/92/commits/d727478033aa5d35d3770ca51f3dd076216f0570